platform/ui/src/components/simpleDialog/SimpleDialog.styl

```diff
@@ -1,6 +1,10 @@
 @import './../../design/styles/common/button.styl'
 @import './../../design/styles/common/global.styl'
 
+.InputDialog
+  .simpleDialog
+    position: relative
+
```

Comment:
We nest a "dialog" within a "dialog"?

Reply:
I have seen this approach in some React libs where they have extended versions of the component as static props of the component class. It's totally not necessary though.